android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -81,7 +83,12 @@ class DVCClient private constructor(
                                 }
                             }
                         }), URI(config?.sse?.url)).build()
-                            eventSource.start()
```

Comment:
note for tomorrow - when testing i think reading the json isn't working properly - lastModified is always null even when it's present in the messageEvent

Reply:
another note - it seems like restarting the connection is sketchy. it consistently calls the start method, but sometimes it stops triggering updates after it has been paused -- will test more tomorrow